### PR TITLE
Closes #1 | Update WikiANN with other South East Asia languages

### LINKED ISSUE #1: Create dataset loader for WikiANN
Dataloader name: `wikiann/wikiann.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?wikiann

|     Dataset| wikiann |
|-------------|---|
| Description | WikiANN (sometimes called PAN-X) is a multilingual named entity recognition dataset consisting of Wikipedia articles annotated with LOC (location), PER (person), and ORG (organisation) tags in the IOB2 format. This version corresponds to the balanced train, dev, and test splits of Rahimi et al. (2019), which supports 176 of the 282 languages from the original WikiANN corpus. |
| Subsets | id, jv, my, tl, th, vi, km, su |
| Languages | ind, jav, mya, tgl, tha, vie, khm, sun |
| Tasks | Named Entiy Recognition |
| License | Apache license 2.0 (apache-2.0) |
| Homepage | https://github.com/panx27/wikiann |
| HF URL | https://huggingface.co/datasets/wikiann |
| Paper URL | https://aclanthology.org/P19-1015/ |

### PR DESCRIPTION
**Pull request by muhsatrio**
Closes #1

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py`.

### CODE REVIEW

**Review by SamuelCahyawijaya — APPROVED** [review]
Tested with the test script and `datasets.load_dataset()` for `mya`, `khm`, `vie`, 'tgl', and `tha` subsets. LGTM!

File: seacrowd/sea_datasets/wikiann/wikiann.py
Comment (sabilmakbar) [inline seacrowd/sea_datasets/wikiann/wikiann.py:16]: Hi, since the code of `mly` is already deprecated based on [this](https://iso639-3.sil.org/code/mly), should we rectify it to either `msa` or `zlm`? It might cause inconsistencies between SEACrowd and NusaCrowd dataloaders, tho (if possible we shd rectify it in NusaCrowd asw). 

I'm more keen to change into `msa` since it's a Macrolang and we don't really know the region of collections (hence `msa` is safer than `zlm`)

wdyt, @SamuelCahyawijaya?

```suggestion
_LANGUAGES = ["ind", "eng", "jav", "min", "sun", "ace", "msa", "map-bms", "mya", "tgl", "tha", "vie", "khm"]
```

File: seacrowd/sea_datasets/wikiann/wikiann.py
Comment (sabilmakbar) [inline seacrowd/sea_datasets/wikiann/wikiann.py:116]: pls adjust this to the prev comments, for both the values of `LANGUAGES_MAP` and `LANG_CODES`

Comment (muhsatrio) [reply seacrowd/sea_datasets/wikiann/wikiann.py]: adjusted, thank you!

File: seacrowd/sea_datasets/wikiann/wikiann.py
Comment (sabilmakbar) [inline seacrowd/sea_datasets/wikiann/wikiann.py:9]: The variables of `DEFAULT_SEACROWD_VIEW_NAME` and `DEFAULT_SOURCE_VIEW_NAME` aren't being referenced anywhere. Consider removing it or reference it by changing hardcoded values of `source` and `seacrowd_seq`

File: seacrowd/sea_datasets/wikiann/wikiann.py
Comment (sabilmakbar) [inline seacrowd/sea_datasets/wikiann/wikiann.py:86]: you seems to forgot referencing its value
```suggestion
_LICENSE = Licenses.APACHE_2_0.value
```

**Review by sabilmakbar — CHANGES_REQUESTED** [review]
few more nits

File: seacrowd/sea_datasets/wikiann/wikiann.py
Comment (muhsatrio) [reply seacrowd/sea_datasets/wikiann/wikiann.py]: adjusted, thank you!

File: seacrowd/sea_datasets/wikiann/wikiann.py
Comment (muhsatrio) [reply seacrowd/sea_datasets/wikiann/wikiann.py]: adjusted, thank you!

File: seacrowd/sea_datasets/wikiann/wikiann.py
Comment (sabilmakbar) [inline seacrowd/sea_datasets/wikiann/wikiann.py:87]: consider removing this to reduce _URLs complexity
```suggestion
_URLs = {
    "https://s3.amazonaws.com/datasets.huggingface.co/wikiann/1.1.0/panx_dataset.zip",
```

and pls adjust it too where the `_URLs` var is being used

**Review by sabilmakbar — CHANGES_REQUESTED** [review]

File: seacrowd/sea_datasets/wikiann/wikiann.py
Comment (muhsatrio) [reply seacrowd/sea_datasets/wikiann/wikiann.py]: adjusted

**Comment by sabilmakbar** [comment]
Hi, this dataloader looks okay to me after being tested, after my last inquiry has been answered by Samcah, we can merge this PR. Thanks for implementing this dataloader, @muhsatrio!

**Review by sabilmakbar — COMMENTED** [review]
Hi, this dataloader looks okay to me after being tested, after my last inquiry has been answered by Samcah, we can merge this PR. Thanks for implementing this dataloader, @muhsatrio!

**Comment by muhsatrio** [comment]
> Hi, this dataloader looks okay to me after being tested, after my last inquiry has been answered by Samcah, we can merge this PR. Thanks for implementing this dataloader, @muhsatrio!

If ok from your side then approved first please :))

**Comment by sabilmakbar** [comment]
After discussion w/ @SamuelCahyawijaya , we decided to remove `eng` from the list of languages in this dataloader, to conform w/ the definition of eligible languages, although the implementation will be different than NusaCrowd

File: seacrowd/sea_datasets/wikiann/wikiann.py
Comment (sabilmakbar) [inline seacrowd/sea_datasets/wikiann/wikiann.py:13]:
```suggestion
_LANGUAGES = ["ind", "jav", "min", "sun", "ace", "zlm", "map-bms", "mya", "tgl", "tha", "vie", "khm"]
```

File: seacrowd/sea_datasets/wikiann/wikiann.py
Comment (sabilmakbar) [inline seacrowd/sea_datasets/wikiann/wikiann.py:111]:
```suggestion
LANGUAGES_MAP = {"ind": "indonesian", "jav": "javanese", "min": "minangkabau", "sun": "sundanese", "ace": "acehnese", "zlm": "malay", "map_bms": "banyumasan", "mya": "myanmar", "tgl": "tagalog", "tha": "thailand", "vie": "vietnam", "khm": "khmer"}
LANG_CODES = {"ind": "id", "jav": "jv", "min": "min", "sun": "su", "ace": "ace", "zlm": "ms", "map_bms": "map-bms", "mya": "my", "tgl": "tl", "tha": "th","vie": "vi","khm": "km"}
```

**Review by sabilmakbar — CHANGES_REQUESTED** [review]

**Comment by muhsatrio** [comment]
> After discussion w/ @SamuelCahyawijaya , we decided to remove `eng` from the list of languages in this dataloader, to conform w/ the definition of eligible languages, although the implementation will be different than NusaCrowd

adjusted, thank you @sabilmakbar

**Review by sabilmakbar — APPROVED** [review]
LGTM! Re-tested after eng is removed! Squash & Merging

**Pull request merged**
